src/main/java/org/example/Trokut.java

```diff
@@ -0,0 +1,21 @@
+package org.example;
+
+public class Trokut extends GeometrijskiLik{
+    double baza;
+    double visina;
+
+    public Trokut(String boja, double baza, double visina){
+        super(boja);
+        this.baza = baza;
+        this.visina = visina;
+    }
+    @Override
+    public double getPovrsina() {
+        return baza * visina;
+    }
+
+    @Override
+    public String getBoja() {
+        return "Boja trokuta je: " + super.getBoja();
```

Comment:
Vidiš, tu mi se malo obilo o glavu insistiranje da fieldovi budu "private". S obzirom da je field boja private u nad-klasi, ti ovdje tom fieldu ne možeš direktno pristupiti (private znači vidi ga samo ta klasa, u ovom slučaju vidljiv je samo klasi GeometrijskiLik). Zato si tu ispravno koristio metodu getBoja iz nad-klase.

Možda je tu bilo pametnije staviti da je boja protected. To znači da je vidljiva klasama koje nasljeđuju tu klasu. Onda bi tu mogao direktno koristiti field boja, a ne super.getBoja()

No, za sad je bitno da shvatiš kako radi ako staviš private (što vidim da jesi) i kako bi radilo kad bi stavio protected. A što ćemo u stvarnom životu raditi, do toga ćemo doći kasnije.

Reply:
da to onda ostavim kao private ili da ga modificiram na protectd ali u komentaru ostavim stari način

Reply:
Oboje je ok, ostavi sada ovako kako je pls.

Reply:
dogovoreno